lib/txmgr/config.go

```diff
@@ -191,8 +191,12 @@ type Config struct {
 }
 
 // NewConfig - creates a new txmgr config from the given CLI config and private key. This is taken and modified from op.
-func NewConfig(ctx context.Context, cfg CLIConfig,
-	privateKey *ecdsa.PrivateKey, client *ethclient.Client) (Config, error) {
+func NewConfig(
```

Comment:
best not change subjective style, then we will get into a ping pong where you change it, I change it back, etc... Take it as it lies.